package.json

```diff
@@ -64,6 +65,7 @@
     "lodash": "4.17.21",
     "native-base": "2.13.8",
     "parse": "2.14.0",
+    "prompt": "1.1.0",
```

Comment:
@jamccomb92 make sure this is in `devDependencies`. Libraries in `dependencies` get bundled and built with the app we deploy to the stores. Libraries in `devDependencies` are libraries for development like linting, testing, releasing etc

Reply:
@hopetambala moved to devDependencies, but got a linting error saying it should be in dependencies.

Reply:
oh weird okay i'll check this out